packages/app-utils/src/components/ConnectionBootstrap.tsx

```diff
@@ -1,12 +1,13 @@
 import React, { useCallback, useEffect, useState } from 'react';
-import { LoadingOverlay } from '@deephaven/components';
+import { BasicModal } from '@deephaven/components';
 import {
   ObjectFetcherContext,
   sanitizeVariableDescriptor,
   useApi,
   useClient,
 } from '@deephaven/jsapi-bootstrap';
-import type { dh } from '@deephaven/jsapi-types';
+import dh from '@deephaven/jsapi-shim';
+import type { dh as DhType } from '@deephaven/jsapi-types';
```

Comment:
`@deephaven/jsapi-shim` shouldn't be used here. It's an old glue package that we should basically never be using (definitely not adding uses of) in DHC.

Previously `dh` was a globally provided module when you connected to the server. We used the shim which is just `export dh = window.dh`. This is still the case, but we have moved to `dh` being a module from the server that doesn't have to be global.

Instead here, use `api` from the `useApi` hook. That is the JS API and the same as `dh`.